src/app/routes/__tests__/Notification.test.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable import/no-named-as-default */
+/* eslint-disable import/named */
+import React from 'react';
+import { shallow } from 'enzyme';
+import NotificationComponent from '../Notification';
+
+const renderDropDown = (args) => {
+  const props = { ...args };
+  return shallow(<NotificationComponent {...props} />);
+};
+
+const Wrapper = renderDropDown();
+const div = Wrapper.find('div');
+
+describe('<DropDown />', () => {
```

Comment:
@harerakalex The first parameter of the describe block is the name of the component you're testing. You shouldn't include self-closing elements like this '<DropDown />'